### PR TITLE
new nextjs error page

### PR DESCRIPTION
**Pull request by elevchyt**
Improved the default error page of nextjs.

image

### CODE REVIEW

File: apps/web/messages/nl.json [hunk 1]
```diff
@@ -2,6 +2,11 @@
   "generic": {
     "error": "Fout"
   },
+  "errors": {
+    "errorPageTitle": "Something bad just happened...",
+    "errorPageDescription": "Our servers could not handle your request. Don't worry, our development team was already notified. Try refreshing the page.",
+    "refreshHint": "Refresh the page"
+  },
```
Comment (trixobird) [inline apps/web/messages/nl.json:9]:
```suggestion
  "errors": {
    "errorPageTitle": "Er is iets fout gegaan...",
    "errorPageDescription": "Onze servers konden uw verzoek niet verwerken. Maak je geen zorgen, ons team is al op de hoogte gebracht. Probeer de pagina te vernieuwen.",
    "refreshHint": "Pagina hernieuwen"
  },
```

File: apps/web/messages/el.json [hunk 2]
```diff
@@ -2,6 +2,11 @@
   "generic": {
     "error": "Σφάλμα"
   },
+  "errors": {
+    "errorPageTitle": "Er is iets fout gegaan...",
+    "errorPageDescription": "Onze servers konden uw verzoek niet verwerken. Maak je geen zorgen, ons team is al op de hoogte gebracht. Probeer de pagina te vernieuwen.",
+    "refreshHint": "Pagina hernieuwen"
+  },
```
Comment (trixobird) [inline apps/web/messages/el.json:9]:
```suggestion
  "errors": {
    "errorPageTitle": "Μόλις συνέβη κάτι κακό...",
    "errorPageDescription": "Οι διακομιστές μας δεν μπορούσαν να χειριστούν το αίτημά σας. Μην ανησυχείτε, η ομάδα ανάπτυξης μας έχει ήδη ειδοποιηθεί. Δοκιμάστε να ανανεώσετε τη σελίδα..",
    "refreshHint": "Ανανέωση σελίδας"
  },
```

File: apps/web/src/components/misc/server-error-page/server-error-page.module.scss
Comment (trixobird) [inline apps/web/src/components/misc/server-error-page/server-error-page.module.scss:1]: why not tailwind? :P :P :P

File: apps/web/src/components/misc/server-error-page/server-error-page.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+'use client';
```
Comment (trixobird) [inline apps/web/src/components/misc/server-error-page/server-error-page.tsx:1]: can we make this on server, and just have the btn as a client component?

File: apps/web/src/components/misc/server-error-page/illustration.tsx
Comment (trixobird) [inline apps/web/src/components/misc/server-error-page/illustration.tsx:1]: do we need this?

File: apps/web/src/components/misc/server-error-page/server-error-page.tsx (hunk 3 above)
Comment (elevchyt) [reply apps/web/src/components/misc/server-error-page/server-error-page.tsx:1]: Nope, error.tsx must be a client component always.
https://nextjs.org/docs/app/building-your-application/routing/error-handling

File: apps/web/src/components/misc/server-error-page/server-error-page.module.scss
Comment (elevchyt) [reply apps/web/src/components/misc/server-error-page/server-error-page.module.scss:1]: It's an edited version of one of mantines [pre-built components](https://ui.mantine.dev/) that come with css modules, like some other pre-made components I've used.
We'd have to install TW and translate the layout to TW, which is unnecessary atm.

**Review by trixobird — APPROVED** [review]

**Pull request merged**